tests/functional/server_configuration/test_server_configuration.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+""" server_configuration function test.
+
+WARNING!!!
+This is a generated test skeleton remove the parts not required by the tests.
+WARNING!!!
+
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+import base64
+import os
+import unittest
+
+from shared.ttypes import RequestFailed
+
+from libtest import env
+
+
+class ConfigTests(unittest.TestCase):
+
+    _ccClient = None
+
+    def setUp(self):
+        """
+        WARNING!!!
+        This is an example how to get the configurations needed by the tests.
+        WARNING!!!
+        """
+
+        # TEST_WORKSPACE is automatically set by test package __init__.py .
+        self._test_workspace = os.environ['TEST_WORKSPACE']
+
+        test_class = self.__class__.__name__
+        print('Running ' + test_class + ' tests in ' + self._test_workspace)
+
+        # Setup a viewer client to test viewer API calls.
+        self._cc_client = env.setup_viewer_client(self._test_workspace)
+        self.assertIsNotNone(self._cc_client)
+
+    def test_edit_admin_notification(self):
```

Comment:
I would split the use cases to separate test functions:
- server without authentication
- server with authentication (only superuser can edit it)
- non ascii characters